runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -330,4 +342,29 @@ public void onNpcDespawned(NpcDespawned npcDespawned)
 			log.debug("Minnow spot {} despawned", npc);
 		}
 	}
+
+	@Subscribe
+	public void onVarbitChanged(VarbitChanged event)
+	{
+		if (!config.trawlerNotification() || client.getGameState() != GameState.LOGGED_IN)
+		{
+			return;
+		}
+
+		int regionID = client.getLocalPlayer().getWorldLocation().getRegionID();
+
+		if ((regionID == 7499 || regionID == 8011)
```

Comment:
Magic numbers :(

Reply:
I've got an issue for centralizing region numbers, until then I think it'll be fine

Reply:
If I had a $1 for every time someone tried to sneak something in by saying they had a fix for it in another commit, and then not fixing it - I'd be a very rich dev.

Reply:
You should at least make named constant at top of the file

Reply:
Uh I'm not exactly trying to sneak something in, I just didn't think it was necessary. If it's that big a deal I'll add some constants :\

Reply:
We do it at other places like that already so I would just follow that for now

Reply:
Done.